VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { ClipboardService } from 'ngx-clipboard';
+import { VhoQueryService } from '../services/vho-query-service.service';
+
+@Component({
+    selector: 'app-copy-quick-link',
+    templateUrl: './copy-quick-link.component.html',
+    styleUrls: ['./copy-quick-link.component.scss']
+})
+export class CopyQuickLinkComponent implements OnInit {
+    @Input() conferenceId: string;
+    hearingId: string;
+    tooltip: string;
+
+    constructor(
+        private clipBoardService: ClipboardService,
+        private vhoQueryService: VhoQueryService,
+        private translateService: TranslateService
+    ) {}
+
+    async ngOnInit() {
+        this.tooltip = this.translateService.instant('copy-quick-link.display-text');
+        const response = await this.vhoQueryService.getConferenceByIdVHO(this.conferenceId);
+        this.hearingId = response && response.hearing_id;
+    }
+
+    copyToClipboard() {
+        this.clipBoardService.copyFromContent(`${this.getbaseUrl()}/quickjoin/${this.hearingId}`);
+        this.tooltip = this.translateService.instant('copy-quick-link.tooltip-copied');
+    }
+
+    getbaseUrl() {
+        return window.location.origin;
```

Comment:
Any chance of this being effected by vanity url or anything like that? How is this done in admin web?

Reply:
In admin web we have a config variable that stores the video web url, do you mean we need that? I am not sure about the vanity url?

Reply:
This will work as long as the video web site its on has the correct url, I dont think we need to introduce a config variable to store the video web url